plantdescriptionengine/src/main/java/eu/arrowhead/core/plantdescriptionengine/consumedservices/orchestrator/dto/ProviderSystem.java

```diff
@@ -0,0 +1,21 @@
+package eu.arrowhead.core.plantdescriptionengine.consumedservices.orchestrator.dto;
+
+import se.arkalix.dto.DtoReadableAs;
+import se.arkalix.dto.DtoToString;
+import se.arkalix.dto.DtoWritableAs;
+
+import static se.arkalix.dto.DtoCodec.JSON;
+
+/**
+ * Data Transfer Object (DTO) interface for provider systems.
+ */
+@DtoReadableAs(JSON)
+@DtoWritableAs(JSON)
+@DtoToString
+public interface ProviderSystem {
```

Comment:
I'm not sure if you need this here, but in Service Registry systems have authenticationInfo and metadata fields as well.

Reply:
The ProviderSystem class has been removed.